core/aws_ddk_core/stages/sqs_lambda.py

```diff
@@ -113,6 +123,18 @@ def __init__(
         else:
             raise ValueError("'code' and 'handler' or 'lambda_function' must be set to instanstiate this stage")
 
+        if create_alarm:
+            self._cloudwatch_alarms = [
```

Comment:
Perhaps we should stick to one alarm for now?
Mostly because with multiple alarms, `evaluation_period` and `threshold` might be different for each alarm